401/array-reverse/__tests__/array-reverse.test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const {describe} = require ('yargs');
+const arrayReverse = require('../array-reverse.js');
+
+
+describe('testing the array reverse module', () => {
+  it('should return an array with reversed elements', () => {
+    const reversedArray = arrayReverse ([1,2,3,4,5,6]);
+    expect(reversedArray).toBe([6,5,4,3,2,1]);
```

Comment:
Great job getting this test written, I would suggest looking back at this because there is a way to do it without .push and that would be great knowledge to have!